esx_service/utils/auth_api.py

```diff
@@ -559,22 +559,29 @@ def _tenant_rm(name, remove_volumes=False):
         error_info = generate_error_info(ErrorCode.TENANT_NOT_EXIST, name)
         return error_info
 
+    error_info, auth_mgr = get_auth_mgr_object()
+
+    if error_info:
+        return error_info
+
     # check if vms that are a part of this tenant have any volumes mounted.
     # If they have, can't delete the tenant.
     if tenant.vms:
         logging.info("_tenant_rm. VMs in tenant are %s", tenant.vms)
 
+        # in multinode mode, we cannot find VM residing on another host
+        # so a gracefull way to admin know about the error
```

Comment:
Need to fix the inline comment something like `in multinode mode, we cannot find VM residing on another host so stopping tenant deletion gracefully`